content/en/docs/Writing policies/jmespath.md

```diff
@@ -408,6 +408,100 @@ spec:
 </p>
 </details>
 
+### Sum
+
+<details><summary>Expand</summary>
+
+<p>
+
+The `sum()` filter very simply adds a list of elements and returns a result. The exception to this rule is `sum()`, which is described here. The official JMESPath library does not have the most fundamental arithmetic operators, such as add, subtract, multiply, and divide. `Sum()`, which is defined in our own JMESPath filters, is beneficial as a streamlined filter when a list of any type of values is needed to be summed, but [`sum()`](https://jmespath.org/specification.html#sum) in the JMESPath library is handy in that it accepts an array of integers as an input. See how sum() differs from the add() filter, which is used to add the values of two fields, in this case. If you want to sum the values in a list of fields, use sum() instead.
```

Comment:
This is perhaps a bit too lengthy and we can shorten:

"The `sum()` filter is a customized version of the default filter present in [upstream JMESPath](https://jmespath.org/specification.html#sum) which brings, in addition to the support for scalars (integers), the ability to sum duration and quantities. `sum()` is similar to `add()` with the difference that `sum()` accepts an array as an input while `add()` does not.

Reply:
It would also be nice to show a simple example, as we do in most other filters, on how to quickly prove this out with the CLI that displays the inputs and actual outputs.

Reply:
The other filters show examples only via policy not via cmd, could you please provide an example where we have used the simple example?

Reply:
Look [here](https://kyverno.io/docs/writing-policies/jmespath/#items).